vm-setup/roles/v1aX_integration_test/tasks/verify.yml

```diff
@@ -17,6 +17,16 @@
     delay: 20
     until: provisioned_bmh.stdout_lines ==  ["node-0", "node-1"]
 
+  - name: Wait until both machines become running.
+    shell: |
+        kubectl get machines -n metal3 -o json | jq -r '.items[]
+        | select (.status.Phase == "Running")
```

Comment:
I think it should be small ''p'' not capital in `.status.Phase`

Reply:
good catch!